src/Core/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -421,7 +421,7 @@ private string MakeOutputColumns(List<LabelledColumn> columns, string columnPref
         /// </summary>
         private string Build(LabelledColumn column, string columnPrefix)
         {
-            if (columnPrefix.IsNullOrEmpty())
+            if (string.IsNullOrEmpty(columnPrefix))
             {
```

Comment:
If we now use the string.IsNullOrEmpty() method, do we still needthe CollectionUtils class method added in this commit: [AuthN Config V2 -> HotReload Aware Authentication settings in dev mod… · Azure/data-api-builder@e777330](https://github.com/Azure/data-api-builder/commit/e77733068d6947aadde540938f47160ce6a486fe)
If we dont need it, lets remove that class in this PR.

Reply:
there were instances in our codebase where `.IsNullorEmpty()` was appended to collections which don't natively provide such an extension method. At one point, an update started using IdentityModel.Tokens extenion method to compare collections and that wasn't supposed to be a public method. They updated that method to be internal so we needed a replacement not to make too many extensive changes to existing code. That's where collectionUtils came in: just copying into our code the extension method to check whether a collection is null or empty. 

So `string.IsNullOrEmpty()` can be used for string, but we still have collections attempting to use that extension.